src/typings/index.d.ts

```diff
@@ -0,0 +1,33 @@
+import './charts/bar-chart'
+import './charts/brush-chart'
+import './charts/bullet-chart'
+import './charts/donut-chart'
+import './charts/grouped-bar-chart'
+import './charts/heatmap-chart'
+import './charts/legend-component'
+import './charts/line-chart'
+import './charts/mini-tooltip'
+import './charts/scatter-plot'
+import './charts/sparkline-chart'
+import './charts/stacked-area'
+import './charts/stacked-bar-chart'
+import './charts/tooltip'
+import './helpers/colors'
+import './helpers/constants'
+
+export * from './charts/bar-chart';
+export * from './charts/brush-chart';
+export * from './charts/bullet-chart';
+export * from './charts/donut-chart';
+export * from './charts/grouped-bar-chart';
+export * from './charts/heatmap-chart';
+export * from './charts/legend-component';
+export * from './charts/line-chart';
+export * from './charts/mini-tooltip';
+export * from './charts/scatter-plot';
+export * from './charts/sparkline-chart';
+export * from './charts/stacked-area';
+export * from './charts/stacked-bar-chart';
+export * from './charts/tooltip';
+export * from './helpers/colors';
+export * from './helpers/constants';
```

Comment:
why do you need to import first?

Reply:
I am also curious what's the point of this

Reply:
It seems it wasn't necessary!